mdakits/solvation-analysis/metadata.yaml

```
@@ -0,0 +1,58 @@
+# Required entries
+## str: name of the project (the respository name)
+project_name: solvation-analysis
+## List(str): a list of authors (or a link to the authors file)
+authors:
+  - orionarcher
+  - hmacdope
+  - laurlee
+  - IAlibay
+## List(str): a list of maintainers
+maintainers:
+  - orionarcher
+  - hmacdope
+## str: a free form description of the mdakit
+description:
+    A comprehensive tool for analyzing liquid solvation structure.
+## List(str): a list of keywords which describe the mdakit
+keywords:
+  - chemistry
+  - electrolytes
+  - solvation structure
+## str: the license the mdakit falls under
+license: GPL-2.0-or-later
+## str: the link to the project's code
+project_home: https://github.com/MDAnalysis/solvation-analysis
+## str: the link to the project's documentation
+documentation_home: https://solvation-analysis.readthedocs.io/en/latest/
+## str: the type of documentation available [UserGuide, API, README]
+documentation_type: UserGuide + API
+
+# Optional entries
+## List(str): a list of commands to use when installing the latest
+## release of the code. Note: only one installation method can currently
+## be defined. We suggest using conda/mamba where possible.
+install:
+  - mamba install -c conda-forge solvation_analysis "pandas<2"
```

See: https://github.com/MDAnalysis/solvation-analysis/issues/93

Ideally if you can either pin your recipees or fix the bug & do a release that would be great, we probably can't have the pandas pin in the install tagline.

---

@orionarcher please review
- updates in the Python package: PR https://github.com/MDAnalysis/solvation-analysis/pull/92
- update to the conda recipe: https://github.com/conda-forge/solvation_analysis-feedstock/pull/8
Then we can remove the pin here and get this done.

---

Once the other PRs have been addressed, we remove the pin from here
```suggestion
  - mamba install -c conda-forge solvation_analysis
```
I hope that the conda-forge PR is reviewed in time so that cf can build an updated package before our Thu 9/21 deadline.